src/JWTGuard.php

```diff
@@ -181,8 +189,15 @@ public function logout($forceForever = false)
     {
         $this->requireToken()->invalidate($forceForever);
 
+        // If we have an event dispatcher instance, we can fire off the logout event
+        // so any further processing can be done. This allows the developer to be
+        // listening for anytime a user signs out of this application manually.
+        $this->events->dispatch(new Logout($this->name, $this->user));
```

Comment:
It feels inconsistent when all other events are wrapped in dedicated `protected fire*Event` methods but this one isn't.

My suggestion:
- remove the comment (same reason as with the other comment)
- add a dedicated `fireLogoutEvent`

Reply:
done